clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangeCoordinate.java

```diff
@@ -96,11 +85,16 @@ protected VeniceChangeCoordinate(String topic, PubSubPosition pubSubPosition, In
 
   public static String convertVeniceChangeCoordinateToStringAndEncode(VeniceChangeCoordinate veniceChangeCoordinate)
       throws IOException {
-    veniceChangeCoordinate.writeExternal(_objectOutputStream.get());
-    _objectOutputStream.get().flush();
-    _objectOutputStream.get().close();
```

Comment:
Was this doing something bad?  I had figured we'd use member variables here to help with GC

Reply:
Yes,` _byteArrayOutputStream.get().toByteArray();` will keep the previously saved value and just appended new value. For example: https://stackoverflow.com/questions/35450584/is-it-possible-to-clear-bytearrayoutputstream. Since we only use this in VeniceChangelogCheckpointThread which runs every 20 seconds, it shouldn't cause obvious performance impact.